docs/SUPPORT.md

```diff
@@ -0,0 +1,15 @@
+<!--
+Copyright (c) 2022 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the GPL, Version 3.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.gnu.org/licenses/gpl-3.0.txt
+-->
+
+# Support
+
+  * To report any issue, create an issue [here](https://github.com/ansible-collections/dellemc.enterprise_sonic/issues).
```

Comment:
They're all the same link. How about we just say
`To report any defects, request functionality, or provide general feedback to the development team, please open an Issue using the [Github issue tracker] (https://github.com/ansible-collections/dellemc.enterprise_sonic/issues).`

Reply:
Addressed as suggested

Reply:
@jeff-yin : Kindly review and confirm.